x-pack/plugins/actions/server/usage/task.ts

```diff
@@ -140,5 +144,5 @@ export function telemetryTaskRunner(
 }
 
 function getNextMidnight() {
-  return moment().add(1, 'd').startOf('d').toDate();
+  return moment().add(1, 'm').startOf('m').toDate();
```

Comment:
Should revert this :)

Reply:
haha, fixed. Thanks for catching it.

Reply:
> For the actions telemetry, I'm seeing both the connector type id and the rule type id in the results. For example, I currently have a rule with a server log action. The telemetry looks like:
> 
> ```
> avg_actions_run_duration_by_connector_type_per_day: {
>   __server-log: 814815,
>   example.always-firing: 814815
> },
> count_connector_types_by_action_run_outcome_per_day: {
>   __server-log: {
>     success: 54
>   },
>   example.always-firing: {
>     success: 54
>   }
> },
> ```
> 
> I think `example.always-firing` shouldn't be there. Is that right?

Is this from the integration test or from an instance you ran locally?
I also saw this in integration tests and thought that somehow rule id is used for a test connector.
Because this is not happening when i run and test Kibana locally.

Reply:
I saw this when running locally with an `example.always-firing` rule with server log connector

Reply:
Solved with https://github.com/elastic/kibana/pull/140943/commits/b9c8252ea77f2b3b3aca77b30c4b3748f1c954df
Forgot to filter saved objects by type...